src/header/css/header-apps.pcss

```diff
@@ -270,6 +270,7 @@ html[dir="rtl"] .dropdown-menu.dropdown-right {
 	text-shadow: 0 1px 8px var(--mono-40);
 	transform: translate3d(0, 0, 0);
 	width: 48px;
+	white-space: nowrap;
 }
```

Comment:
I think you should also add

  overflow: hidden;
  text-overflow: ellipsis;

Reply:
Good catch!  Those styles have been added, let me know if there are any other changes needed. Thanks!